lib/src/container/encapsulate.rs

```diff
@@ -71,24 +73,31 @@ fn build_oci(
     let commit_meta = &commit_v.child_value(0);
     let commit_meta = glib::VariantDict::new(Some(commit_meta));
 
+    let mut ctrcfg = oci_image::Config::default();
+    let mut imgcfg = oci_image::ImageConfiguration::default();
+    let labels = ctrcfg
+        .labels_mut()
+        .get_or_insert_with(|| Default::default());
+    let mut manifest = ocidir::new_empty_manifest().build().unwrap();
+
     if let Some(version) =
         commit_meta.lookup_value("version", Some(glib::VariantTy::new("s").unwrap()))
     {
         let version = version.str().unwrap();
-        writer.add_config_annotation("version", version);
-        writer.add_manifest_annotation("ostree.version", version);
+        labels.insert("version".into(), version.into());
     }
 
-    writer.add_config_annotation(OSTREE_COMMIT_LABEL, commit);
-    writer.add_manifest_annotation(OSTREE_COMMIT_LABEL, commit);
```

Comment:
I dropped the manifest annotation bit here - it isn't making it into Docker schema, so it's not useful.